instance/tests/models/test_openedx_secret_key_mixins.py

```diff
@@ -80,6 +81,16 @@ def test_secret_key_settings(self):
         for to_var, from_var in OPENEDX_SHARED_KEYS.items():
             self.assertEqual(secret_key_settings[to_var], secret_key_settings[from_var])
 
+    def test_secret_key_settings_no_key(self):
+        """
+        Test that secret key settings are empty if not master key is set.
```

Comment:
@pomegranited I must have been tired by the time I wrote this (or my English got impacted by writing too much code).